bin/bazel_to_go.py

```diff
@@ -1,75 +1,200 @@
 #!/usr/bin/env python
 
+#
```

Comment:
It would be nice to unify this sort of script across all the repos. As it is now, mixer, manager, and istio all generate some version istio/api protobufs in slightly different ways. Is this something that could be abstracted away in a *.bzl in some common repo, e.g. istio/api?